lib/src/day_view/_internal_day_view_page.dart

```diff
@@ -0,0 +1,218 @@
+              timeLineWidth: timeLineWidth,
+            ),
+          CustomPaint(
+            size: Size(width, height),
+            painter: HourLinePainter(
+              lineColor: hourIndicatorSettings.color,
+              lineHeight: hourIndicatorSettings.height,
+              offset: timeLineWidth + hourIndicatorSettings.offset,
+              minuteHeight: heightPerMinute,
+              verticalLineOffset: verticalLineOffset,
+              showVerticalLine: showVerticalLine,
+            ),
+          ),
+          Align(
+            alignment: Alignment.centerRight,
+            child: Container(
+              height: height,
+              width: width - timeLineWidth - hourIndicatorSettings.offset,
+              child: Stack(
+                children: _generateEvents(),
+              ),
+            ),
+          ),
+          Align(
+            key: ValueKey(heightPerMinute),
+            child: _buildTimeLine(),
+            alignment: Alignment.centerLeft,
+          ),
+        ],
+      ),
+    );
+  }
+
+  List<Widget> _generateEvents() {
+    List<OrganizedCalendarEventData<T>> events = eventArranger.arrange(
+        controller.getEventsOnDay(date),
+        height,
+        width - timeLineWidth - hourIndicatorSettings.offset,
+        heightPerMinute);
+
+    return List.generate(events.length, (index) {
+      return Positioned(
+        top: events[index].top,
+        bottom: events[index].bottom,
+        left: events[index].left,
+        right: events[index].right,
+        child: eventTileBuilder(
+          date,
+          events[index].events,
+          Rect.fromLTWH(
+              events[index].left,
+              events[index].top,
+              width - events[index].right - events[index].left,
+              height - events[index].bottom - events[index].top),
+          events[index].startDuration,
+          events[index].endDuration,
+        ),
+      );
+    });
+  }
+}
+
+class _LiveTimeIndicator extends StatefulWidget {
+  final double width;
+  final double height;
+  final double timeLineWidth;
+  final HourIndicatorSettings liveTimeIndicatorSettings;
+  final double heightPerMinute;
+
+  const _LiveTimeIndicator(
+      {Key key,
+      this.width,
+      this.height,
+      this.timeLineWidth,
+      this.liveTimeIndicatorSettings,
+      this.heightPerMinute})
+      : super(key: key);
+
+  @override
+  _LiveTimeIndicatorState createState() => _LiveTimeIndicatorState();
+}
+
+class _LiveTimeIndicatorState extends State<_LiveTimeIndicator> {
+  Timer _timer;
+  DateTime _currentDate;
+
+  @override
+  void initState() {
+    super.initState();
+
+    _currentDate = DateTime.now();
+    _timer = Timer(Duration(minutes: 1), setTimer);
+  }
+
+  @override
+  void dispose() {
+    _timer?.cancel();
+    super.dispose();
+  }
+
+  void setTimer() {
+    if (mounted)
+      setState(() {
+        _currentDate = DateTime.now();
+        _timer = Timer(Duration(minutes: 1), setTimer);
```

Comment:
This won't be accurate if you start at half a min or so. Maybe we could improve it. Find out other ways of doing this.